changelogs/fragments/235-add-win_zip.yml

```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
```

Comment:
Changelog fragments for new modules aren't needed. It is automatically added when the changelog is generated.

Reply:
Thanks! I will remove it then